platform_plugin_turnitin/conf/locale/es_419/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,305 @@
+"you and your teammates below. Be sure to add\n"
+"                                  descriptions to your files to help your "
+"teammates identify them.\n"
+"                              "
+msgstr ""
+
+#: templates/turnitin/oa_response.html:261
+msgid "We could not upload files"
+msgstr ""
+
+#: templates/turnitin/oa_response.html:267
+msgid "We could not delete files"
+msgstr ""
+
+#: templates/turnitin/oa_response.html:274
+msgid "Select one or more files to upload for this submission."
+msgstr ""
+
+#: templates/turnitin/oa_response.html:276
+msgid "Select a file to upload for this submission."
+msgstr ""
+
+#: templates/turnitin/oa_response.html:280
+msgid "Supported file types: "
+msgstr ""
+
+#: templates/turnitin/oa_response.html:285
+msgid "Upload files"
+msgstr ""
+
+#: templates/turnitin/oa_response.html:287
+msgid "Upload file"
+msgstr ""
+
+#: templates/turnitin/oa_response.html:304
+msgid "This is a team submission."
+msgstr ""
+
+#: templates/turnitin/oa_response.html:306
+msgid ""
+"One team member should submit a response with the team’s shared files and a "
+"text response on behalf of the entire team."
+msgstr ""
+
+#: templates/turnitin/oa_response.html:308
+msgid ""
+"One team member should submit a response with the team’s shared files on "
+"behalf of the entire team."
+msgstr ""
+
+#: templates/turnitin/oa_response.html:310
+msgid ""
+"One team member should submit a text response on behalf of the entire team."
+msgstr ""
+
+#: templates/turnitin/oa_response.html:312
+msgid "One team member should submit on behalf of the entire team."
+msgstr ""
+
+#: templates/turnitin/oa_response.html:314
+msgid ""
+"\n"
+"                  Learn more about team assignments here: (<a "
+"target=\"_blank\" href=\"https://support.edx.org/hc/en-us/"
+"articles/360000191067-Submit-your-"
+"response#h_01FVD8SXM9E5H2DNAG87X25ZHR\">link</a>)\n"
+"                  "
+msgstr ""
+
+#: templates/turnitin/oa_response.html:322
+msgid "We could not submit your response"
+msgstr ""
+
+#: templates/turnitin/oa_response.html:328
+msgid ""
+"\n"
+"                          The response text and files related to your "
+"response will be sent to Turnitin for a plagiarism check.\n"
+"                          Please read <a href=\"https://www.turnitin.com/"
+"agreement.asp\" target=\"_blank\">Turnitin's EULA</a>\n"
+"                          before submitting, as by doing so, you are "
+"explicitly accepting the agreement.\n"
+"                        "
+msgstr ""
+"\n"
+"                          La respuesta de texto y los archivos relacionados "
+"con su respuesta se enviarán a Turnitin para una verificación de plagio.\n"
+"                          Por favor, lea el <a href=\"https://www.turnitin.com/"
+"agreement.asp\" target=\"_blank\">EULA de Turnitin</a>\n"
+"                          antes de enviar, ya que al hacerlo, está aceptando "
+"explícitamente el acuerdo.\n"
+"                        "
```

Comment:
so this works just adding this translation and no other?

Reply:
Yes, according to local tests it is not necessary to add the other translations.